lib/gds-components/date-input/DateInput.tsx

```diff
@@ -10,7 +10,7 @@ export interface DateInputProps {
 }
 
 export const DateInput = ({ value, label, name, onChange }: DateInputProps) => {
-  const [month = "", year = ""] = value.split("-");
+  const [year = "", month = ""] = value.split("-");
```

Comment:
I have swapped these 2 around as the date is in the following format when both year and month are present: YYYY-MM